.env

```
@@ -48,7 +48,7 @@ OKP4_SMART_CONTRACT_COGNITARIUM_CODE_ID='6'
 OKP4_CHAIN_ADDRESS='okp41r0pf2d78w8w29sm9a6qm8x6yqshezm0k6vwcrg'
 GRAPHQL_URI='https://api.nemeton.staging.okp4.network/graphql'
 OKP4_KEPLR_WALLET_EXTENSION_URL='https://chrome.google.com/webstore/detail/keplr/dmkamcknogkgcdfhhbddcghachkejeap'
-OKP4_EXPLORER_WALLET_URL='https://explore.okp4.network/wallet/import'
+OKP4_EXPLORER_WALLET_URL='https://explore.okp4.network/OKP4%20testnet'
```

It's not about `wallet` anymore, you can update the variable name
```suggestion
OKP4_EXPLORER_DASHBOARD_URL='https://explore.okp4.network/OKP4%20testnet'
```